lib/oxttl/src/line_formats.rs

```diff
@@ -54,8 +54,8 @@ impl RuleRecognizer for NQuadsRecognizer {
     fn recognize_next(
         mut self,
         token: N3Token<'_>,
-        context: &mut NQuadsRecognizerContext,
-        results: &mut Vec<Quad>,
+        context: &mut Self::Context,
+        results: &mut Vec<Self::Output>,
```

Comment:
This looks less readable to me than the concrete types because one would need to look at the associated type to know what the argument actually are. I am a bit not at ease with it. May you remove this part from your MR?

Reply:
yeah, i wasn't too certain about this one either - in some cases concrete types are better,  will revert

Reply:
Thank you!